global.json

```diff
@@ -1,14 +1,15 @@
 {
   "sdk": {
-    "version": "6.0.300",
+    "version": "7.0.100-preview.4.22252.9",
```

Comment:
I don't think this repo should build with .net 7. We want to keep this repo using the latest supported LTS version.

Reply:
The purpose of upgrading to .NET 7 here is because in order for my Arcade PR to pass validation, Maestro.Tasks needs to have net7.0 output assemblies. Shouldn't it not be an issue since .NET 7 tooling can build all the other projects with a target framework of net6.0? Leaving some [of these hardcoded paths ](https://github.com/lbussell/arcade/blob/a881adf79c45db4d8107e4cc766512768db2829e/src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/PublishBuildAssets.proj#L33)to reference net6.0 assemblies is another option, if you want to keep everything in this repo targeting LTS only.

@alexperovich any input here?